build failed with gcc 11.2.0 compiler and glibc 2.35

In file included from /home/u/ipu6/ipu6-camera-hal/ipu6-camera-hal/include/api/ICamera.h:76,
                 from /home/u/ipu6/ipu6-camera-hal/ipu6-camera-hal/src/hal/ICamera.cpp:19:
/home/u/ipu6/ipu6-camera-hal/ipu6-camera-hal/include/api/Parameters.h:2545:56: error: ‘size_t’ has not been declared
 2545 |     int setLensShadingMap(const float *lensShadingMap, size_t lensShadingMapSize);
      |                                                        ^~~~~~
/home/u/ipu6/ipu6-camera-hal/ipu6-camera-hal/include/api/Parameters.h:2555:51: error: ‘size_t’ has not been declared
 2555 |     int getLensShadingMap(float **lensShadingMap, size_t &lensShadingMapSize) const;
      |                                                   ^~~~~~
make[2]: *** [CMakeFiles/camhal_static.dir/build.make:790: CMakeFiles/camhal_static.dir/src/hal/ICamera.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:312: CMakeFiles/camhal_static.dir/all] Error 2
make: *** [Makefile:156: all] Error 2


Try to add diff below can make it build successfully.
diff --git a/include/api/Parameters.h b/include/api/Parameters.h
index 5808252..e2d7e77 100644
--- a/include/api/Parameters.h
+++ b/include/api/Parameters.h
@@ -76,6 +76,7 @@
 #include <vector>
 #include <set>
 #include <stdint.h>
+#include \<cstddef\>

--- a/include/api/Parameters.h
+++ b/include/api/Parameters.h
@@ -76,6 +76,7 @@
 #include <vector>
 #include <set>
 #include <stdint.h>
+#include <cstddef>
 
 namespace icamera {
 

